### PR TITLE
Class player

### PR DESCRIPTION
**Pull request by ninaprus**

### CODE REVIEW

File: class_player.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        self._height = height
+        self._weight = weight
+        self._team = None
+        Player.instance_count +=1
+
+    def __str__(self):
+       return f'This is {self._name}'
+
+    def get_age(self):
+        print(f'{self._name} age is {self._age}')
```
Comment (mmykhailenko) [inline class_player.py:17]: the method should return something, `return self._age`

File: class_player.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+    def get_height(self):
+        print(f'{self._name} height is {self._height}')
```
Comment (mmykhailenko) [inline class_player.py:20]: return self._height

File: class_player.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+    def get_weihgt(self):
+        print(f'{self._name} weight is {self._weight}')
```
Comment (mmykhailenko) [inline class_player.py:23]: return self._weight

File: class_player.py [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+# Имел атрибуты экземпляра player, name.
+#4) реализовать метод который бы выводил всех игроков команды. Team().name
+
+class Team:
+    name = {}
+
+    def __init__(self, team_name, *players):
+        self.team_name = team_name
+        self._all_team = []
+        self.player = self.new_team(*players)
```
Comment (mmykhailenko) [inline class_player.py:36]: self.player change to self.players

File: class_player.py [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+    def get_weihgt(self):
+        print(f'{self._name} weight is {self._weight}')
+
+
+#3) Реализовать класс Команда, который бы имел возможность добавлять нового игрока в команду.
+# Имел атрибуты экземпляра player, name.
+#4) реализовать метод который бы выводил всех игроков команды. Team().name
+
+class Team:
+    name = {}
```
Comment (mmykhailenko) [inline class_player.py:31]: not sure why we need this Classs attribute here

File: class_player.py [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+        self._all_team = []
+        self.player = self.new_team(*players)
+
+    def new_team(self, *players):
+        for player in players:
+            if isinstance(player, Player):
+                self.player = player
+                player._team = self.team_name
+                self._all_team.append(self.player)
+                Team.name[self.team_name] = self.get_players_team()
```
Comment (mmykhailenko) [inline class_player.py:44]: no sense

**Review by mmykhailenko — APPROVED** [review]

File: class_player.py (hunk 1 above)
Comment (ninaprus) [reply class_player.py:17]: и print, и return? так всегда нужно?

**Pull request merged**